caffe2/CMakeLists.txt

```diff
@@ -1076,6 +1076,14 @@ if(USE_XPU)
         "-Wl,--whole-archive,\"$<TARGET_FILE:torch_xpu_ops>\" -Wl,--no-whole-archive")
     endif()
   endif()
+
+  # Set exported torch-xpu-ops header to torch
+  set(INCLUDE_DIR_FOR_TORCH "${CMAKE_BINARY_DIR}/aten/src/ATen/xpu" )
+  file(MAKE_DIRECTORY ${INCLUDE_DIR_FOR_TORCH})
+  file(GLOB xpu_h "${TORCH_ROOT}/third_party/torch-xpu-ops/src/aten/*.h")
+  foreach(HEADER  ${xpu_h})
+    file(COPY ${HEADER} DESTINATION ${INCLUDE_DIR_FOR_TORCH})
```

Comment:
You might not copy them but append TORCH_XPU_OPS_INCLUDE_DIR into caffe2/ATen include dirs. Please check cmake of dynamo should depends on caffe2/ATen include dirs. Then sources in dynamo could find the header by nature without copy.

Reply:
I designed it this ‘copy' way because I wanted the include path to be consistent with cuda/cpu (`<ATen/xpu/EmptyTensor.h>`) so that it would be more readable.